packages/convex-helpers/server/zod4.functions.test.ts

```diff
@@ -258,6 +267,12 @@ describe("zCustomQuery, zCustomMutation, zCustomAction", () => {
       >();
     });
 
+    test("zCustomQuery with no args", async () => {
+      const t = convexTest(schema, modules);
+      const response = await t.query(testApi.testQueryNoArgs);
```

Comment:
another way we can test the behavior of it 
```suggestion
      const response = await t.run((ctx) => ctx.runQuery(testApi.testQueryNoArgs));
```
Since `ctx.runQuery` also expects no args to make args optional